scalableminds/webknossos
# Make mappings field in datasource optional

Mappings field is not used by wk itself anymore. to preserve backwards compatibility, this PR makes it optional, rather than removing it entirely.

### URL of deployed dev instance (used for testing):
- https://mappingsoptional.webknossos.xyz

### Steps to test:
- Remove mappings attribute from a segmentation layer in datasource-properties.json
- wk should still serve the dataset, mappings should still work
- edit page should work without error message, changing bbox and saving should still work

### Issues:
- fixes #4511

------
